bitcoinplusplus/layer-2/lightning-on-taproot.md

```diff
@@ -1,402 +1,491 @@
-Even though, to be quite frank, today already the odds of multiple hop All right.
-Even today, the odds of multiple hops being correlated on chain are fairly low because you have to have multiple channels go on chain with HDLCs in flight, which, let's be honest, is not a big risk.
-What this does help though with is wormhole attacks, for example, where somebody can see, if somebody has multiple nodes that they're controlling and those nodes are talking to one another and it just so happens that multiple of those nodes are involved in the same route of payment, then they would know that they are involved in the same payment and they would be able to steal some money.
-With BTLCs, they wouldn't know because there is no correlation whatsoever.
-So how do BTLCs work precisely?
-Actually who here already knows how BTLCs work?
-All right, cool.
-I'm really glad that I'm finally able to actually bring up something that people here are not as familiar with.
+
+But That being said, another aspect where we can somewhat improve privacy as PTLCs, because of the issue that I mentioned earlier, where payments in hops could not be correlated.
+Even though, to be quite frank, today already the odds of multiple hops.
+
+Even today, the odds of multiple hops being correlated on-chain are fairly low because you have to have multiple channels go on-chain with HTLCs in-flight, which, let's be honest, not a big risk.
+What this does help though with is wormhole attacks, for example, where somebody can see. If somebody has multiple nodes that they're controlling and those nodes are talking to one another and it just so happens that multiple of those nodes are involved in the same route of payment, then they would know that they are involved in the same payment and they would be able to steal some money.
+With PTLCs, they wouldn't know because there is no correlation whatsoever.
+
+### HOW TO PTLCS WORK
+
+So how do PTLCs work precisely?
+Actually who here already knows how PTLCs work?
+All right, cool. I'm really glad that I'm finally able to actually bring up something that people here are not as familiar with.
+
 Let's say that Alice is trying to pay Emily, or more precisely, Emily is trying to get paid by Alice.
 I'm using this particular phrasing because Emily is the one that has to initiate the whole flow by creating an invoice.
 Her invoice is no longer going to be a hatch.
 What Emily does is she generates some random number z, some number of which is within the finite field we're using for our elliptic curve, and she multiplies it with a generator point, as we always do, so essentially the public point corresponding to that private integer, and she sends uppercase Z to Alice.
-So that uppercase Z is now the invoice.
-So the invoice, instead of being a hash, is and will be curve point.
+
+So that uppercase Z is now the invoice.So the invoice, instead of being a hash, is and will be curve point.
```

Comment:
Those letter are part of math equations so they should be in backticks. Also, on the transcript we don't need to include "uppercase" or "lowercase" as we get that from the writing.

Reply:
equations need attention to make sure that they represent the actual content of the presentation.